app/models.py

```diff
@@ -119,16 +125,16 @@ def serialize(self):
             'name': self.name,
             'slug': self.slug,
             'framework': self.framework,
-            'countersignerName': (
-                self.framework_agreement_details or {}
-            ).get("countersignerName"),
-            'frameworkAgreementVersion': (
-                self.framework_agreement_details or {}
-            ).get("frameworkAgreementVersion"),
+            'countersignerName': (self.framework_agreement_details or {}).get("countersignerName"),
+            'frameworkAgreementVersion': (self.framework_agreement_details or {}).get("frameworkAgreementVersion"),
             'variations': (self.framework_agreement_details or {}).get("variations", {}),
             'status': self.status,
             'clarificationQuestionsOpen': self.clarification_questions_open,
             'lots': [lot.serialize() for lot in self.lots],
+            'application_close_date': (
+                self.application_close_date.strftime(DATETIME_FORMAT) if self.application_close_date else ''
```

Comment:
None of our other nullable datetime fields do this (empty string if no date) - they seem to just return a null if there's no date.  I'm not saying that's the right thing, but there's a benefit in being consistent how we treat null things.